src/test/unit/parsers/test_event_parser.py

```diff
@@ -264,3 +264,143 @@ def test_parse_event_api_gw_v2():
+                                "Test": {"Type": "String", "Value": "TestString"},
+                                "TestBinary": {"Type": "Binary", "Value": "TestBinary"},
+                            },
+                            "MessageId": "95df01b4-ee98-5cb9-9903-4c221d41eb5e",
+                        }
+                    },
+                    {
+                        "Sns": {
+                            "Message": "Hello from SNS2!",
+                            "MessageAttributes": {
+                                "Test": {"Type": "String", "Value": "TestString"},
+                                "TestBinary": {"Type": "Binary", "Value": "TestBinary"},
+                            },
+                            "MessageId": "95df01b4-ee98-5cb9-9903-4c221d41eb5e",
+                        }
+                    },
+                ]
+            }
+        )
+    )
+
+
+def test_parse_event_sqs():
+    not_order_sns_event = {
+        "Records": [
+            {
+                "messageId": "059f36b4-87a3-44ab-83d2-661975830a7d",
+                "receiptHandle": "AQEBwJnKyrHigUMZj6rYigCgxlaS3SLy0a...",
+                "body": "Test message1",
+                "attributes": {
+                    "ApproximateReceiveCount": "1",
+                    "SentTimestamp": "1545082649183",
+                    "SenderId": "AIDAIENQZJOLO23YVJ4VO",
+                    "ApproximateFirstReceiveTimestamp": "1545082649185",
+                },
+                "messageAttributes": {"a": 1},
+                "md5OfBody": "e4e68fb7bd0e697a0ae8f1bb342846b3",
+                "eventSource": "aws:sqs",
+                "eventSourceARN": "arn:aws:sqs:us-east-2:123456789012:my-queue",
+                "awsRegion": "us-east-2",
+            },
+            {
+                "messageId": "2e1424d4-f796-459a-8184-9c92662be6da",
+                "receiptHandle": "AQEBzWwaftRI0KuVm4tP+/7q1rGgNqicHq...",
+                "body": "Test message2",
+                "attributes": {
+                    "ApproximateReceiveCount": "1",
+                    "SentTimestamp": "1545082650636",
+                    "SenderId": "AIDAIENQZJOLO23YVJ4VO",
+                    "ApproximateFirstReceiveTimestamp": "1545082650649",
+                },
+                "messageAttributes": {"b": 2},
+                "md5OfBody": "e4e68fb7bd0e697a0ae8f1bb342846b3",
+                "eventSource": "aws:sqs",
+                "eventSourceARN": "arn:aws:sqs:us-east-2:123456789012:my-queue",
+                "awsRegion": "us-east-2",
+            },
+        ]
+    }
+
+    order_sns_event = EventParser.parse_event(event=not_order_sns_event)
+
+    assert json.dumps(order_sns_event) == json.dumps(
+        OrderedDict(
+            {
+                "Records": [
+                    {
+                        "body": "Test message1",
+                        "messageAttributes": {"a": 1},
+                        "messageId": "059f36b4-87a3-44ab-83d2-661975830a7d",
```

Comment:
I think that the `SentTimestamp` is important here too,
still not mandatory :P

Reply:
Same as above